zebedee-cms/src/test/java/com/github/onsdigital/zebedee/model/CollectionTest.java

```diff
@@ -45,30 +45,16 @@
 import java.io.OutputStream;
 import java.io.Writer;
 import java.net.URI;
-import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Date;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Can you update your setting to stop these wildcard imports please:
https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports